learn/tanssi/templates.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Templates Included in Tanssi
```

Comment:
If I think about what templates means I have no clue.

So would change "Templates" for Appchain Templates or something like that